client/ts/package.json

```diff
@@ -27,12 +27,15 @@
     "@synnaxlabs/freighter": "workspace:*",
     "@synnaxlabs/x": "workspace:*",
     "async-mutex": "^0.5.0",
+    "nanoid": "^5.0.7",
+    "uuid": "^10.0.0",
```

Comment:
Why do we need this as a dependency? Also, the "@synnaxlabs/x" package has an `id` module that we can use to generate unique IDs.